src/main/java/core/basesyntax/service/ReportGenerator.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.service;
+
+import java.util.List;
+import java.util.Map;
+
+public interface ReportGenerator {
+    List<String> getReport(Map<String, Integer> fruitRepository);
```

Comment:
your storage imitates a real database. Don't pass it like a parameter. Create a class with a static map 
```
public class Storage {
    public static final Map<String, Integer> STORAGE = new HashMap<>();
}
```